packages/hub/src/graphql/mutations/adminRebuildSearchIndex.ts

```diff
@@ -0,0 +1,22 @@
+import { builder } from "@/graphql/builder";
+
+import { rebuildSearchableTable } from "../helpers/searchHelpers";
+
+builder.mutationField("adminRebuildSearchIndex", (t) =>
+  t.withAuth({ signedIn: true }).field({
+    description: "Admin-only query for rebuilding the search index",
```

Comment:
This will be necessary after the initial release; after that, the index will be maintained automatically unless we make a mistake somewhere (see `indexFooId` methods in `searchHelpers.ts`).

Reply:
That's neat. I look forward to future admin dashboards, but it's a minor thing.

Reply:
Done: https://quri-hub-git-search-quantified-uncertainty.vercel.app/admin

<img width="514" alt="Captura de pantalla 2023-12-09 a la(s) 16 50 55" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/10b4e0e5-84e5-4d8c-902a-bcc4fa0a7fd4">